python/py-pytest-mpl/Portfile

```diff
@@ -1,39 +1,61 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           python 1.0
 PortGroup           github 1.0
+PortGroup           python 1.0
 
-github.setup        matplotlib pytest-mpl 0.9 v
-name                py-${name}
+github.setup        matplotlib pytest-mpl 0.11 v
+revision            0
+name                py-${github.project}
 categories-append   devel
+
 license             BSD
+maintainers         nomaintainer
```

Comment:
Please don't remove me as the maintainer. (Feel free to add yourself as a co-maintainer, if you like.)

Reply:
Whoops, unintentional I created this from scratch simply because I missed that it already exists. I dropped that commit, so this port is no longer part of the PR.